index.html

```diff
@@ -91,15 +89,19 @@ <h3>How do you compare?</h3>
 
```

Comment:
On line 58, can you use the same design for <metric form> and <imperial form> so it will look more consistent?

Reply:
solved in [001c905](https://github.com/leemoispace/UdacityDinosaurs/pull/1/commits/001c905e77f2367b37693a245588b8c33a47ce34)